build.gradle

```diff
@@ -100,6 +103,7 @@ configurations {
     compile {
         extendsFrom fastutil
     }
+    log4j
```

Comment:
@Meronat Do we actually need another sourceset to handle the `Log4j2Plugins` data file? Was it not handled properly before?